OutOfSchool/OutOfSchool.WebApi/Middlewares/AuthorizationTokenMiddleware.cs

```diff
@@ -2,7 +2,7 @@
 
 public class AuthorizationTokenMiddleware
 {
-    private static readonly string[] Hubs = new string[] { "/chathub", "/notificationhub" };
+    private static readonly string[] Hubs = new string[] { "/hubs/chat", "/hubs/notification" };
```

Comment:
(Code style) It looks like these string constants are reused. So they could be extracted from the list, made public and reused.